src/test/scala/com/mpc/scalats/configuration/FieldNamingSpec.scala

```diff
@@ -1,9 +1,10 @@
 package com.mpc.scalats.configuration
 
-import org.scalatest.{ FlatSpec, Matchers }
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
 import org.scalatest.prop.TableDrivenPropertyChecks._
 
-final class FieldNamingSpec extends FlatSpec with Matchers {
+final class FieldNamingSpec extends AnyFlatSpec with Matchers {
```

Comment:
Why?

Reply:
Due to version bump of scalatest to 3.10.0